src/wallet/rpcwallet.cpp

```diff
@@ -5520,17 +5520,15 @@ UniValue z_mergetoaddress(const UniValue& params, bool fHelp)
                 isToTaddr = true;
             },
             [&](libzcash::SaplingPaymentAddress addr) {
-                isToSaplingZaddr = true;
                 // If Sapling is not active, do not allow sending to a sapling addresses.
                 if (!saplingActive) {
                     throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid parameter, Sapling has not activated");
                 }
+                estimatedTxSize += OUTPUTDESCRIPTION_SIZE;
             },
             [](libzcash::SproutPaymentAddress) { },
-            [](libzcash::UnifiedAddress) {
-                throw JSONRPCError(
-                        RPC_INVALID_PARAMETER,
-                        "Invalid parameter, unified addresses are not yet supported.");
```

Comment:
This should probably have an entry in the release notes? Looks like it didn't in the original PR.

Reply:
Added
```
- The `z_mergetoaddress` RPC method can now merge _to_ UAs and can also send
  between different shielded pools (when `AllowRevealedAmounts` is specified).
```
but might hold onto it until the RC PR … modulo the comments I made above.

Reply:
Pushed and started build.